src/gamestruct/displayable.cpp

```diff
@@ -10,6 +10,10 @@ Displayable::Displayable(Position current, char display_char) :
         position_=new Changeable<Position>(current);
     }
 
+void Displayable::setDisplayChar(char display_char) {
+    displayChar_ = display_char;
```

Comment:
bisogna mettere un changable in displayChar_ così la volta dopo che verrà chiamata rendere fare un force